slim/nets/resnet_v2.py

```diff
@@ -25,6 +25,8 @@
 
 The key difference of the full preactivation 'v2' variant compared to the
 'v1' variant in [1] is the use of batch normalization before every weight layer.
+Another difference is that 'v2' ResNets do not include an activation function in
```

Comment:
@derekjchow we actually removed this comment in a previous PR. See https://github.com/tensorflow/models/pull/1465.